spec/traits.dd

```diff
@@ -411,6 +413,33 @@ static assert(!__traits(isTemplate,foo!int()));
 static assert(!__traits(isTemplate,"string"));
 ---
 
+$(H2 $(GNAME isModule))
+
+        $(P Takes one argument. If that argument is a symbol that refers to a module then
+        $(D true) is returned, otherwise $(D false). Packages that contain a `package.d`
+        file are also considered to be modules.
+        )
+
+---
+import std.algorithm.sorting;
+static assert(!__traits(isModule, std));
+static assert(__traits(isModule, std.algorithm));
+static assert(__traits(isModule, std.algorithm.sorting));
+---
+
+$(H2 $(GNAME isPackage))
+
+        $(P Takes one argument. If that argument is a symbol that refers to a package
+        then $(D true) is returned, otherwise $(D false).
+        )
+
+---
+import std.algorithm.sorting;
+static assert(__traits(isPackage, std));
+static assert(__traits(isPackage, std.algorithm));
+static assert(!__traits(isPackage, std.algorithm.sorting));
```

Comment:
How about adding short comments for people not so familiar with the layout of `std.algorithm`?

```
static assert(__traits(isPackage, std)); // no package.d
static assert(__traits(isPackage, std.algorithm)); package.d
static assert(!__traits(isPackage, std.algorithm.sorting)); // module file
```

(similar suggestion for the example above)